### ISSUE TITLE
Doc the blender file

### ISSUE THREAD

**Issue by gilbertohasnofb** [Juanvvc/c172p-detailed#17]
3D15 - Doc the blender file and the caveats involved. Intents to give ludomotico freedom to change the model if i die flying (hehehe).

**Comment by gilbertohasnofb** [comment]
Is this still relevant?

**Comment by Juanvvc** [comment]
Yes it is! There are many layers and groups and materials, and we must use a special plug-in in blender to export the model. If we do not document the .blend file, any modeller after us will have a hard time to understand many things and he/she will probably just import the .ac file and recreate the .blend file.

Thevirtualfer explained the different layers in the forum, and the special plug-in we need is linked somewhere over there. But there are about 100 pages of messages in that thread.

I started to document the .blend file in my PC, but I have neither finished the job nor submitted the initial version.

This is totally related to #16

**Comment by onox** [comment]
You could try to document it in an `.md` file (Markdown) or use the [wiki](https://github.com/Juanvvc/c172p-detailed/wiki) here on Github.

**Comment by wkitty42** [comment]
why not just write a plain text doc file and store it in the directory with the .blend file? that would be the best thing... it doesn't need any fancy window dressing for looks... it is just a technical document, after all :wink:

**Comment by onox** [comment]
`.md` files are plain text.

**Comment by Juanvvc** [comment]
Actually, I'm ok with the .md file. As onox says, it is a text file with some basic formatting. I use that format constantly for documentation and wiki :)

**Comment by wkitty42** [comment]
yes, .md files are plain text but no markup is needed... .md is only an extension anyway ;)

**Comment by legoboyvdlp** [comment]
This becomes more important perhaps as the 3D model becomes more and more advanced.

**Comment by wlbragg** [comment]
Unfortunately I never saw this before and had no idea it even exist. I'm not sure this really applies anymore, at least as far as the layers and groups are concerned. I have serious questions as to the usability of groups  anymore in the .ac. Or how any of that works.
I know I had to assign effects to the individual object that were originally part of a Blender group, unless they are defined in a "name" animation.

**Comment by gilbertohasnofb** [comment]
@wlbragg in that case would you like to close this?

**Comment by wlbragg** [comment]
@gilbertohasnofb Yes and no. As far as I am concerned the special plug-in needed in the past has been overridden by massive changes in almost ever aspect of the models in this project. Some of the last ones I just encountered in the instrument repositioning work. However the part about documenting any nuances with the blends and ac's couldn't hurt. I will go on the record and say that we should in the future not set ourself up to have to rely on special or uncommon tools or plugins to produce the aircraft. My opinion anyway.
So I think my answer to your question is it wouldn't hurt to document the blend and ac nuances but not as it applies to special plugins anymore.
